sagemaker-datawrangler/joined-dataflow/explore_data.ipynb

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
<ul><li>Use present tense as per guidelines (i.e. we download and explore the data)</li><li>I would reword "In this notebook, we will download and explore the data that is used to build the SageMaker DataWrangler flow file for data processing." as "In this notebook, we download and explore the data. To prepare it for a machine learning task, we then process this data by writing a SageMaker DataWrangler .flow file."</li><li>I would reword "After running this notebook, you can follow the <a href="https://github.com/aws/amazon-sagemaker-examples/raw/ecf4ab542d371fe5131106285c27379971875497/sagemaker-datawrangler/joined-dataflow/README.md" target="_blank">README.md</a> for the step by step instructions to build the data processing flow file to prepare data for the machine learning task." to "After running this notebook, you can follow the <a href="https://github.com/aws/amazon-sagemaker-examples/raw/ecf4ab542d371fe5131106285c27379971875497/sagemaker-datawrangler/joined-dataflow/README.md" target="_blank">README.md</a> for step by step instructions how to write the SageMaker DataWrangler .flow file."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>